lib/fog/libvirt/models/compute/templates/server.xml.erb

```diff
@@ -118,6 +119,14 @@
     </console>
     <input type='tablet' bus='usb'/>
     <input type='mouse' bus='ps2'/>
+<% end -%>
+<% if arch == "s390x" -%>
```

Comment:
Could you avoid the extra condition check here and do an `if else`?

Reply:
Asbolutely, good point. I am actually reversing the order so it's more readable: if s390x else.

Reply:
🆒